models/User.model.js

```diff
@@ -3,6 +3,7 @@ const Schema = mongoose.Schema;
 const jwt = require("jsonwebtoken");
 //remove this later and add to env
 const JWT_SECRET_KEY = "jaguar";
```

Comment:
We might consider grabbing this variable from the `.env` file as well. If we don't this won't match with the variable we're [using here](https://github.com/cqnguy23/Hackathon_Project/blob/fbebddd798dd8fbfa8e133631a9e3d34b002317a/middlewares/auth.middleware.js#L3).

Reply:
@ngducn  Next step is put this JWTkey in an .env, then make sure we did not "gitignore" the .env.
so that everyone could use it in this repo. 
We will delete and gitignore it later.